src/api/app/views/webui/request/show.html.haml

```diff
@@ -10,7 +10,11 @@
                                                      number: @bs_request.number,
                                                      diff_to_superseded: @diff_to_superseded,
                                                      superseding: @bs_request.superseding }
-- @pagetitle = "Request #{@bs_request.number}"
+:ruby
```

Comment:
@jcronenberg there is already a `:ruby` block in line 1, you could just move your code to this block to save the additional `:ruby`

Reply:
besides that it looks good to me :+1:

Reply:
Oh true, good point :+1: